### PR TITLE
update french language (+ standardize some format)

### PR DESCRIPTION
**Pull request by abarrau**
1/ Update French tag value for files :
account_lang.php --> done
adif_lang.php --> done
admin_lang.php --> done
awards_lang.php
contesting_lang.php --> done
eqsl_lang.php	--> non change
export_lang.php
filter_lang.php
general_words_lang.php --> in progress
gridsquares_lang.php
lotw_lang.php
menu_lang.php
notes_lang.php --> done
options_lang.php
qslcard_lang.php --> done
qso_lang.php
station_lang.php
statistics_lang.php --> done

2/ addition of a space (on 1 line of the adif file for each language) to standardize the structure of the files

3/ standardize css form-control for 1 field on add/edit user view

### CODE REVIEW

**Comment by HB9HIL** [comment]
Hello @abarrau 

If you want to standardize all the language files I would recommend to use the format 

" [Text with ' ] "

Not 

' [Text with \' ] '

It's easier for unexperienced people to edit the language. 

What do you think?

**Comment by abarrau** [comment]
Hi @HB9HIL,
 
when I look at the files, almost all of them are formatted 'text\'text'.
I can change to "text 'text" by starting with the French files and then generalizing with my tool.
but before I launch into the changes (and receive a refusal), can we validate that this is the normal desired by the team?
THANKS,

**Comment by abarrau** [comment]
1/ state update tranlate french language

DONE (with delimitor ")
- account_lang.php
- adif_lang.php
- admin_lang.php
- contesting_lang.php
- eqsl_lang.php
- export_lang.php
- filter_lang.php
- general_words_lang.php
- gridsquares_lang.php
- lotw_lang.php
- notes_lang.php
- qslcard_lang.php
- statistics_lang.php

IN PROGRESS
- awards_lang.php

TO DO
- menu_lang.php
- options_lang.php
- qso_lang.php
- station_lang.php

2/ change add/edit station_profil for tag "gen_find_zone_xxx" (general_words)
--> same consistency for all languages

3/ add tag "gridsquares_gridsquares_worked" (general_words) for all language file
--> exist only for english

4/ add tag "account_user_mastodon_hint" to views/users/add 
- ok for edit, but not tag for add page

File: application/views/station_profile/edit.php [hunk 1]
```diff
@@ -205,12 +205,12 @@
...
 							?>
 						</select>
-						<small id="stationCQInputHelp" class="form-text text-muted"><?php echo lang("gen_find_zone_cq"); ?></small>
+						<small id="stationCQInputHelp" class="form-text text-muted"><?php echo lang("gen_find_zone_cq_part1")." <a href='https://zone-check.eu/?m=cq' target='_blank'>".lang("gen_find_zone_part2")."</a> ".lang("gen_find_zone_part3"); ?></small>
 					</div>
 
 					<!-- ITU Zone -->
 					<div class="form-group">
-                    	<label for="stationITUZoneInput"><?php echo lang("gen_hamradio_itu_zone"); ?></label>
+                    	<label for="stationITUZoneInput"><?php echo lang("gen_hamradio_itu_zone_part1"); ?></label>
```
Comment (HB9HIL) [inline application/views/station_profile/edit.php:213]: remove "_part1" here

Comment (abarrau) [reply application/views/station_profile/edit.php]: hello, done

File: application/language/french/lotw_lang.php [hunk 2]
```diff
@@ -2,61 +2,62 @@
...
-$lang['lotw_cert_expired'] = 'Au moins un de vos certificats LoTW est expiré !';
+$lang['lotw_cert_expiring'] = "Au moins un de vos certificats LoTW est sur le point d'expirer !";
+$lang['lotw_cert_expired'] = "Au moins un de vos certificats LoTW est expiré !";
 
 // Lotw User
-$lang['lotw_user'] = 'Cette station utilise LoTW.';
-$lang['lotw_last_upload'] = 'Dernier upload';
+$lang['lotw_user'] = "Cette station utilise LoTW";
+$lang['lotw_last_upload'] = "Dernier téléchargement";
+$lang['lotw_report_download_overview_helptext'] = "Cloudlog utilisera le nom d'utilisateur et le mot de passe LoTW stockés dans votre profil utilisateur pour télécharger le journal vers LoTW.<br/> Les téléchargements du journal Cloudlog auront toutes les confirmations depuis la date choisie, ou depuis votre dernière confirmation LoTW (récupérée de votre journal), jusqu'à présent.";
```
Comment (HB9HIL) [inline application/language/french/lotw_lang.php:63]: double see Line 38

Comment (abarrau) [reply application/language/french/lotw_lang.php]: done

**Review by HB9HIL — APPROVED** [review]
Everything seems to work as expected. We should stop here now to avoid getting this PR to big. 
@AndreasK79 Can you have a quick check aswell?

**Pull request merged**